packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -815,14 +815,8 @@ export default class UserStorageController extends BaseController<
         for await (const _ of Array.from({
```

Comment:
Indeed `Array.from` is not async and therefore `for ()` without await would be more accurate and should give the same behavior.

However, it seems that the unused array is constructed with empty entries which are also never used just for the sake of iterating a number of times? Wouldn't a classic for-loop be more idiomatic here? E.g.
```
for (let i=0; i<numberOfAccountsToAdd; i++) {
} 
```
?

Reply:
You're both correct! I didn't use a simple for loop because I was sure that I needed a for await loop in order to be able to loop sequentially while correctly awaiting things in said loop. 
Guess I was traumatized by `Array.forEach` 😅 

Thanks, learnt something today!